docs/source/markdown/podman-system-connection.1.md

```diff
@@ -1,43 +1,34 @@
 % podman-system-connection(1)
 
 ## NAME
-podman\-system\-connection - Record ssh destination for remote podman service
+podman\-system\-connection - Manage ssh destination for podman service(s)
 
 ## SYNOPSIS
-**podman system connection** [*options*] [*ssh destination*]
+**podman system connection** *subcommand*
 
 ## DESCRIPTION
-Record ssh destination for remote podman service(s). The ssh destination is given as one of:
- - [user@]hostname[:port]
- - ssh://[user@]hostname[:port]
+Manage ssh destination(s) for podman service(s).
 
-The user will be prompted for the remote ssh login password or key file pass phrase as required. `ssh-agent` is supported if it is running.
+The user will be prompted for the ssh login password or key file pass phrase as required. `ssh-agent` is supported if it is running.
```

Comment:
```suggestion
The user will be prompted for the ssh login password or key file pass phrase as required. The `ssh-agent` is supported if it is running.
```

Reply:
Day late dollar short, I know (you've already made the change) - but I've never heard anyone use an article to refer to ssh-agent. It looks and sounds super weird to read 'the ssh-agent`. But, maybe that's just me.